distribution/helm/vector-aggregator/values.yaml

```diff
@@ -2,6 +2,9 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# Additional labels to add to all resources
+customLabels: {}
```

Comment:
Can we rename these to be `commonLabels` - I think that's more clear in their purpose.